packages/patterns-react/src/patterns/ListSection/__stories__/ListSection.stories.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { LISTSECTION } from '../../../internal/FeatureFlags';
+import { withKnobs, text, select, object } from '@storybook/addon-knobs';
+import '../../../../../styles/scss/patterns/listsection/index.scss';
+import ListSection from '../ListSection';
+import readme from '../README.md';
+
+if (LISTSECTION) {
```

Comment:
Yes we do. This is here so that a blank story doesn't appear on the non-experimental storybook. We actually should do this for the other experimental stories that are showing up as blank (since those are turning up as null in those stories).

Reply:
Ok, then nevermind. For some reason I thought story book was handling this. Maybe I need to go back and double check some of my own feature flags to make sure I'm following this pattern too. Thanks!